sycl/include/sycl/ext/oneapi/matrix/matrix-jit.hpp

```diff
@@ -451,6 +451,276 @@ class wi_element {
   }
 };
 
+template <size_t NumRows, size_t NumCols, matrix_layout Layout, typename Group>
```

Comment:
add a comment that uint16_t here is representing bf16 type as we have been doing for the previous matrix functions.
Since the AMX and DPAS implementations don't support uint16_t, this should raise no problem. 
Add a comment that the plan is to move towards SYCL bfloat16 as it makes it to the specification.